src/components/Components.tsx

```diff
@@ -97,11 +93,15 @@ class Components extends React.Component<ComponentsGlobalProps, XCSpyState> {
 
     componentDidMount() {
         const props = this.props;
-        // const comps = props.compositionModel.components;
-        const parsers = props.parsers;
         const componentProperties = {};
-        for (let i = 0; i < parsers.length; i++) {
-            const parser = parsers[i];
+        const components = props.compositionModel.components;
+        const parsers = [];
+        components.map((component) => {
+            const parser = new Parser(component);
+            parser.parse();
```

Comment:
As you return nothing in your map function, I think you may need to use forEach instead of map. No ?